src/Makefile

```diff
@@ -1224,16 +1224,16 @@ $(DOTNET_DESTDIR)/$($(2)_NUGET_REF_NAME)/ref/$(DOTNET_TFM)/Microsoft.$(1).xml: $
 
 $($(2)_DOTNET_BUILD_DIR)/doc/Microsoft.$(1).xml: $($(2)_DOTNET_BUILD_DIR)/ref/Microsoft.$(1).xml $($(2)_DOTNET_BUILD_DIR)/ref/Microsoft.$(1).dll build/.build-adr-stamp | $($(2)_DOTNET_BUILD_DIR)/doc
 ifdef ENABLE_XAMARIN
-	$(MAKE) run -C $(MACCORE_PATH)/tools/apple-doc-reader ASSEMBLY="$(abspath $($(2)_DOTNET_BUILD_DIR)/ref/Microsoft.$(1).dll)" INPUT="$(abspath $($(2)_DOTNET_BUILD_DIR)/ref/Microsoft.$(1).xml)" OUTPUT="$(abspath $$@)"
+	$(Q) $(MAKE) run -C $(MACCORE_PATH)/tools/apple-doc-reader ASSEMBLY="$(abspath $($(2)_DOTNET_BUILD_DIR)/ref/Microsoft.$(1).dll)" INPUT="$(abspath $($(2)_DOTNET_BUILD_DIR)/ref/Microsoft.$(1).xml)" OUTPUT="$(abspath $$@)" 1>/dev/null
 else
 	$$(Q) $$(CP) $($(2)_DOTNET_BUILD_DIR)/ref/Microsoft.$(1).xml $$@
 endif
 endef
 
 build/.build-adr-stamp:
 ifdef ENABLE_XAMARIN
-	$(Q) $(MAKE) build -C $(MACCORE_PATH)/tools/apple-doc-reader
-	$(Q) $(MAKE) create -C $(MACCORE_PATH)/tools/apple-doc-reader
+	$(Q) $(MAKE) build -C $(MACCORE_PATH)/tools/apple-doc-reader 1>/dev/null
+	$(Q) $(MAKE) create -C $(MACCORE_PATH)/tools/apple-doc-reader 1>/dev/null
```

Comment:
I think this solution has a downside: passing V=1 to make (`make V=1`) doesn't make it verbose, it's permanently silenced.

IMHO the best solution would be to copy the `mk/quiet.mk` file to the maccore repository, and then use the variables from that file in `maccore/tools/apple-doc-reader/Makefile`.

So building would be something like:

```make
$(EXEDIR)/AppleDocReader: $(ADR_SOURCES) $(ADR_NUGET_CONFIG)
	$(Q) $(DOTNET) build --configfile $(ADR_NUGET_CONFIG) ./AppleDocReader/AppleDocReader.csproj $(DOTNET_BUILD_VERBOSITY)
``

Reply:
@rolfbjarne good idea, made the changes. let me know if it's on par with what you were expecting!